### DISCUSSION TITLE
[기능 추가] 판매자와 구매자간 실시간 채팅 서비스

### DISCUSSION THREAD

**Discussion by wintiger98** [SSapingMall/backend#56] [Ideas]
당근마켓처럼 판매자와 구매자간 실시간 채팅 서비스를 구현하면 거래하기 편리할 것 같습니다. 다들 어떠신가요?
그리고 기술 면에서도 웹소켓을 이용한 실시간 데이터 통신도 경험해보고 MSA를 적용해볼 수 있는 좋은 기회일듯?

**Comment by kkho9654** [comment]
좋아요 근데 상품 거래랑 게시판 추가기능도 해야돼서 회의 한번 하시죠 추가기능 뭐할지
